### PR TITLE
Bump dependencies of OaaL operator to Knative 0.18

### PR DESCRIPTION
**Pull request by markusthoemmes**
Hopefully future bumps will be less invasive. This was very crappy because of all of the K8s 1.18 bumps, but here we go. Picking off one after the other!

/hold

Until we're moving up into 0.18.

### CODE REVIEW

File: openshift-knative-operator/go.mod [hunk 1]
```diff
@@ -3,29 +3,29 @@ module github.com/openshift-knative/serverless-operator/openshift-knative-operat
...
+	google.golang.org/genproto v0.0.0-20200914193844-75d14daec038 // indirect
+	google.golang.org/grpc v1.32.0 // indirect
+	k8s.io/api v0.19.0
 	k8s.io/apimachinery v0.19.1
 	k8s.io/client-go v11.0.1-0.20190805182717-6502b5e7b1b5+incompatible
-	k8s.io/code-generator v0.18.8 // indirect
-	knative.dev/operator v0.17.2
-	knative.dev/pkg v0.0.0-20200831162708-14fb2347fb77
-	knative.dev/test-infra v0.0.0-20200915193842-f4d4232c1f04
+	knative.dev/eventing v0.14.1-0.20200428210242-f355830c4d70 // indirect
```
Comment (nak3) [inline openshift-knative-operator/go.mod:18]: Although it is "indirect", do we need eventing v0.14.1? :thinking:

Comment (markusthoemmes) [reply openshift-knative-operator/go.mod]: Good catch, added eventing to the set of floating deps.

**Review by nak3 — APPROVED** [review]
/lgtm

**Pull request merged**